### DISCUSSION TITLE
Add the symmetric constraint from SolveSpace

### DISCUSSION THREAD

**Discussion by ZanQdo** [hlorus/CAD_Sketcher#220] [Feature Requests]
The [symmetric constraint](https://solvespace.readthedocs.io/en/latest/constraints/symmetric.html) available in SolveSpace is missing from the constraint list in CADSketcher.

This constraint is very useful to symmetrize specific elements in a sketch and should not be confused by Blender's mirror modifier since the modifier will mirror the entire geometry and the constraint allows to symmetrize specific elements.

**Comment by hlorus** [comment]
Agree this would be nice, there seems to be a big in the python solver module which mixes up the symmetric constraint with symmetric-vertical. I will have to report it.

**Reply by cgagner** [reply]
It looks like this was fixed in #523 under issue #508.
